algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/treatment/HasRadiologicalProgressionFollowingLatestTreatmentLineTest.kt

```diff
@@ -0,0 +1,177 @@
+                stopYear = 2025,
+                stopMonth = 9
+            ),
+            TreatmentTestFactory.treatmentHistoryEntry(setOf(TreatmentTestFactory.treatment("2", true)),
+                startYear = 2025,
+                startMonth = 10,
+                stopYear = 2025,
+                stopMonth = 11
+            )
+        )
+        val evaluation = FUNCTION.evaluate(TreatmentTestFactory.withTreatmentHistory(treatments))
+        assertEvaluation(EvaluationResult.PASS, evaluation)
+        assertThat(evaluation.passMessages).containsExactly("Last systemic treatment stopped and radiological progression is assumed.")
+    }
+
+    @Test
+    fun shouldPassWithMustBeRadiological() {
+        val treatments = listOf(
+            TreatmentTestFactory.treatmentHistoryEntry(setOf(TreatmentTestFactory.treatment("1", true)),
+                stopReason = StopReason.TOXICITY,
+                startYear = 2024,
+                startMonth = 10,
+                bestResponse = TreatmentResponse.STABLE_DISEASE,
+                stopYear = 2025,
+                stopMonth = 9
+            ),
+            TreatmentTestFactory.treatmentHistoryEntry(setOf(TreatmentTestFactory.treatment("2", true)),
+                stopReason = StopReason.PROGRESSIVE_DISEASE,
+                bestResponse = TreatmentResponse.PROGRESSIVE_DISEASE,
+                startYear = 2025,
+                startMonth = 10,
+                stopYear = 2026,
+                stopMonth = null
+            )
+        )
+        val evaluation = FUNCTION_PD_MUST_BE_RADIOLOGICAL.evaluate(TreatmentTestFactory.withTreatmentHistory(treatments))
+        assertEvaluation(EvaluationResult.PASS, evaluation)
+        assertThat(evaluation.passMessages).containsExactly("Last systemic treatment resulted in PD")
+    }
+
+    @Test
+    fun unableToDetermineRadiologicalProgressionWhenTreatmentsWithoutStartDate() {
+        val treatments = listOf(
+            TreatmentTestFactory.treatmentHistoryEntry(setOf(TreatmentTestFactory.treatment("1", true)),
+                stopReason = StopReason.TOXICITY,
+                bestResponse = TreatmentResponse.STABLE_DISEASE,
+                startYear = null,
+                startMonth = null
+            ),
+            TreatmentTestFactory.treatmentHistoryEntry(setOf(TreatmentTestFactory.treatment("2", true)),
+                stopReason = StopReason.PROGRESSIVE_DISEASE,
+                bestResponse = TreatmentResponse.PROGRESSIVE_DISEASE,
+                startYear = 2025,
+                startMonth = null
+            )
+        )
+        val evaluation = FUNCTION.evaluate(TreatmentTestFactory.withTreatmentHistory(treatments))
+        assertEvaluation(EvaluationResult.UNDETERMINED, evaluation)
+        assertThat(evaluation.undeterminedMessages).containsExactly("Unable to determine radiological progression following latest treatment line due to treatments without start date.")
+    }
+
+
+    companion object {
```

Comment:
no need to use companion object here. You may see this elsewhere but it's normally due to old code being converted from java to kotlin.

Convention would be to make the objects under test fields of the test class.